src/renderer/lib/slippiBackend.ts

```diff
@@ -100,12 +100,20 @@ export async function fetchPlayKey(): Promise<PlayKey> {
 
   handleErrors(res.errors);
 
+  const connectCode = res.data.getUser?.connectCode?.code;
+  const playKey = res.data.getUser?.private?.playKey;
+  if (!connectCode || !playKey) {
+    // If we don't have a connect code or play key, return this as null such that logic that
+    // handles it will cause the user to set them up.
+    return null;
+  }
+
   return {
     uid: user.uid,
-    connectCode: res.data.getUser.connectCode.code,
-    playKey: res.data.getUser.private.playKey,
-    displayName: res.data.getUser.displayName,
-    latestVersion: res.data.getLatestDolphin.version,
+    connectCode: connectCode,
+    playKey: playKey,
+    displayName: res.data.getUser?.displayName,
+    latestVersion: res.data.getLatestDolphin?.version,
```

Comment:
```suggestion
    connectCode,
    playKey,
    displayName,
    latestVersion,
```

This repo uses object shorthand notation so we should stick with that.

Reply:
Addressed by https://github.com/project-slippi/slippi-launcher/pull/275/commits/fdabb249215b917821f9657d80dd191b3831b81d